weed/topology/topology.go

```diff
@@ -35,6 +35,7 @@ type Topology struct {
 
 	volumeSizeLimit  uint64
 	replicationAsMin bool
+	VacuumStatus     bool
```

Comment:
rename this

Reply:
How about renaming it  to "DisableVacuum"

Reply:
Done